src/BenchmarkDotNet.Diagnostics.Windows/Sessions.cs

```diff
@@ -15,6 +15,21 @@
 
 namespace BenchmarkDotNet.Diagnostics.Windows
 {
+    internal class HeapSession : Session
+    {
+        public HeapSession(DiagnoserActionParameters details, EtwProfilerConfig config, DateTime creationTime)
+            : base(FullNameProvider.GetBenchmarkName(details.BenchmarkCase) + "Heap", details, config, creationTime)
+        {
+        }
+
+        protected override string FileExtension => ".userheap.etl";
```

Comment:
nit: lack of new line

```suggestion
        protected override string FileExtension => ".userheap.etl";

```